src/app/app.component.ts

```diff
@@ -1,22 +1,27 @@
 import { Component, OnInit } from '@angular/core';
 import { LanguageService } from './i18n/language.service';
 import { NavigationEnd, Router } from '@angular/router';
+import { TitleAndMetaTagsService } from './service/title-meta-tags/title-and-meta-tags.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent implements OnInit {
+export class AppComponent implements OnInit  {
 
   constructor(
     private languageService: LanguageService,
-    private router: Router
+    private titleAndMetaTagsService: TitleAndMetaTagsService,
+    private router: Router,
   ) {}
 
   ngOnInit(): void {
```

Comment:
Please remove type (it is not necessary for hooks)

Reply:
fixed